tests/test_conversationhandler.py

```diff
@@ -1126,6 +1219,29 @@ def slowbrew(_bot, update):
         assert handler.conversations.get((self.group.id, user1.id)) is None
         assert self.is_timeout
 
+    def test_conversation_timeout_warning_only_shown_once(self, recwarn):
+        ConversationHandler(
+            entry_points=self.entry_points,
+            states={
+                self.THIRSTY: [
+                    ConversationHandler(
+                        entry_points=self.entry_points,
+                        states={
+                            self.BREWING: [CommandHandler('startCoding', self.code)],
+                        },
+                        fallbacks=self.fallbacks,
+                    )
+                ]
+            },
+            fallbacks=self.fallbacks,
+            conversation_timeout=100,
+        )
+        assert len(recwarn) == 1
```

Comment:
This test doesn't test too much, if there is only 1 child conversaton in the first place ;)

Reply:
Oops you're right, gonna fix that